pkg/ipamd/ipamd.go

```diff
@@ -692,9 +718,19 @@ func (c *IPAMContext) increaseDatastorePool(ctx context.Context) {
 		return
 	}
 	// Try to add more Cidrs to existing ENIs first.
+	if c.inInsufficientCidrCoolingPeriod() {
+		log.Debug("Recently we had InsufficientCidr Error hence will wait for 2 mins before retrying")
```

Comment:
nit: `Error` -> `error`.
2 mins should be replaced with `%v` and  insufficientCidrErrorCooldown